packages/wp-now/src/download.ts

```diff
@@ -2,33 +2,57 @@ import fs from 'fs-extra';
 import path from 'path';
 import https from 'https';
 import unzipper from 'unzipper';
+import os from 'os';
 import { IncomingMessage } from 'http';
 import {
+	DEFAULT_WORDPRESS_VERSION,
 	SQLITE_PATH,
 	SQLITE_URL,
 	WORDPRESS_VERSIONS_PATH,
-	WP_DOWNLOAD_URL,
 	WP_NOW_PATH,
 } from './constants';
 
+export function isValidWordpressVersion(version: string): boolean {
+	const versionPattern =
```

Comment:
> I created the Regex based on the releases on https://wordpress.org/download/releases/#betas

this would be a part of a great comment here. we might want to move this into the module scope and give it its own docblock, with an explanation, some example version strings, and a link to the releases page where the regex came from.

this will give someone more insight when something goes wrong and they need to fix or change it.

Reply:
I've moved isValidWordpressVersion` to a new folder:
https://github.com/WordPress/wordpress-playground/pull/252/commits/206e52ab824e95d15e9d723c7d7a2777f4940251

And I've created this issue https://github.com/WordPress/wordpress-playground/issues/261 to refactor the function into a new package, as suggested here https://github.com/WordPress/wordpress-playground/pull/252#discussion_r1186470038